command/build.py

```diff
@@ -1,8 +1,39 @@
+import yaml as pyYaml
+import command.identify as identify
+
 class BuildCmd():
     name = 'build'
-
-    def __init__(self):	
+    
```

Comment:
Remove unnecessary whitespace

Reply:
@SamiSousa What is unnecessary about this whitespace? It makes the pace between setting name and the init method more readable.

Reply:
The newline is fine, but it's followed by 4 spaces which are unnecessary.

Reply:
Addressed in latest commit.